packages/System.Text.Encoding.CodePages.4.0.1/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,31 @@
+This Microsoft .NET Library may incorporate components from the projects listed
```

Comment:
Not for this specific issue.

The System.Text.Encoding.CodePages package was listed in the packages.config (not sure which part of the program actually requires it), and, thus, it was fetched automatically at compilation time.
I can see, that you include all downloaded NuGet packages in the repository, thus, I have included this one as well in order to follow the same order.

On a side note, I don't know if it is necessary to include required packages since those will be resolved/downloaded during the compilation (at least in Windows/Visual Studio environment).
Maybe we can look through the list of packages and remove unnecessary ones, though it will require additional tests.

Reply:
Sorry, I am not that experienced in collaborative development. Maybe I should have opened an issue for this, rather than making changes. Please, excuse my "baby action".

Reply:
I think is fine. We are all learning. For us is about C#. If you think some of the libraries are not needed and they can be downloaded during the compilation time. Just remove then from your PR and this will clean the Repo.

Reply:
Sorry, for the late reply - I have to deal with quite a lot of other stuff last days.
I am not sure if those packages are necessary to compile under Linux or MacOS, for example. If so then it is better to preserve them in the repo.
I cannot figure out if I can cancel my PR, thus, I think I will just make a new one that will only update the important parts, and the question about the packages should be moved to another discussion.
Does it sound fine?

Reply:
Yes, please. Can you only update the file needed?